src/main/webapp/customerDataAngularJS.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script
+    src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.4/angular.js"></script>
+ 
+<title>Spring Boot AngularJS example</title>
+<script type="text/javascript">
```

Comment:
Could this be moved to it's own separate file and be imported? For consistency? 

For a reference, I do it here: https://github.com/chriswbluer/Pokedex/blob/main/webapp/src/main/webapp/WEB-INF/views/pokedexManagement.jsp#L155